src/libraries/Microsoft.PowerFx.Core/Entities/External/IExternalDataSource.cs

```diff
@@ -19,6 +19,9 @@ internal interface IExternalDataSource : IExternalEntity, IExternalPageableSymbo
 
         bool RequiresAsync { get; }
 
+        // !!! Change this in PA
+        bool IsClearable { get; }
```

Comment:
>IsClearable [](http://example.com/codeflow?start=13&length=11)

This is not used anywhere in this PR. Needed?

Reply:
This is going to be used by Clear/ClearCollect functions. I'll remove it for this function and add it back when needed.